packages/components/src/components/hive-footer.tsx

```diff
@@ -0,0 +1,253 @@
+import { ComponentProps, ReactNode } from 'react';
+import { cn } from '../cn';
+import { HiveCombinationMark } from '../logos';
+import { PRODUCTS } from '../products';
+import { IFooterExtendedProps, ILink } from '../types/components';
+import { Anchor } from './anchor';
+import {
+  CSAStarLevelOneIcon,
+  DiscordIcon,
+  GitHubIcon,
+  LinkedInIcon,
+  TwitterIcon,
+  YouTubeIcon,
+} from './icons';
+
+export interface HiveFooterProps extends IFooterExtendedProps {}
+
+export function HiveFooter({ className, logo, resources = [], sameSite }: HiveFooterProps) {
+  return (
+    <footer className={cn('relative flex justify-center px-4 py-6 xl:px-[120px]', className)}>
+      <div className="mx-4 grid w-full max-w-[75rem] grid-cols-1 gap-x-6 text-green-800 max-lg:gap-y-16 sm:grid-cols-4 lg:gap-x-8 xl:gap-x-10 dark:text-neutral-400">
+        <div className="max-lg:col-span-full">
+          <Anchor href="https://the-guild.dev" sameSite={sameSite} {...logo}>
+            <HiveCombinationMark className="h-8 w-auto text-green-1000 dark:text-white" />
+          </Anchor>
+          <p className="mt-6 lg:mt-8">Open-source GraphQL management platform</p>
+        </div>
+        <div className="col-span-full grid grid-flow-row grid-cols-2 justify-stretch gap-6 text-sm sm:col-span-4 sm:grid-cols-3 lg:col-span-3 lg:pb-12 lg:text-base">
+          <List heading="Products" links={products} />
+          <div className="flex flex-col gap-[inherit]">
+            <List heading="Developer" links={DEVELOPER} />
+            {ENTERPRISE.length > 0 && <List heading="Enterprise" links={ENTERPRISE} />}
+            {resources.length > 0 && <List heading="Resources" links={resources} />}
+          </div>
+          <div className="flex flex-col gap-[inherit]">
+            <List heading="Company" links={COMPANY} />
+            <a
+              href="https://the-guild.dev/graphql/hive#pricing"
+              className="-m-2 p-2 font-medium hover:text-blue-700 hover:underline dark:hover:text-blue-100"
+            >
+              Pricing
+            </a>
+            <a
+              className="-m-2 p-2 font-medium hover:text-blue-700 hover:underline dark:hover:text-blue-100"
+              href="https://the-guild.dev/contact"
+              onClick={event => {
+                if (typeof window !== 'undefined' && '$crisp' in window) {
+                  (window.$crisp as { push(cmd: string[]): void }).push(['do', 'chat:open']);
+                  event.preventDefault();
+                }
```

Comment:
```suggestion
                  (window.$crisp as { push(cmd: string[]): void }).push(['do', 'chat:open']);
                  event.preventDefault();
```

onClick is not available on server